plugins/permissions/src/guilds/welcomeScreen.ts

```diff
@@ -1,6 +1,16 @@
 import { BotWithCache } from "../../deps.ts";
 import { requireBotGuildPermissions } from "../permissions.ts";
 
+export function getWelcomeScreen(bot: BotWithCache) {
+  const getWelcomeScreenOld = bot.helpers.getWelcomeScreen;
+
+  bot.helpers.getWelcomeScreen = function (guildId) {
+    requireBotGuildPermissions(bot, guildId, ["MANAGE_GUILD"]);
```

Comment:
Closes: #1973 

## If the welcome screen is not enabled, the `MANAGE_GUILD` permission is required.

@Skillz4Killz Is there a way to check if the welcome screen is enabled or not? Because we need to check `MANAGE_GUILD` permission ONLY IF THE WELCOME SCREEN IS DISABLED, otherwise you can still get it WITHOUT the `MANAGE_GUILD` permission